plugins/modules/launchdarkly_custom_role.py

```diff
@@ -13,58 +13,58 @@
 DOCUMENTATION = r"""
 ---
 module: launchdarkly_custom_role
-short_description: Manage LaunchDarkly Custom Roles
+short_description: Manage custom roles
 description:
-    - Manager LaunchDarkly Custom Roles
+    - Manage LaunchDarkly custom roles. To learn more about custom roles, read L(Custom roles, https://docs.launchdarkly.com/home/members/custom-roles).
```

Comment:
For docs-reviewers: The `L(name, url)` format is how something is rendered as a link for these modules.